test/decompose.spec.js

```diff
@@ -0,0 +1,102 @@
+/* global describe, it, expect */
+import {
+  decompose,
+  compose,
+  identity,
+  translate,
+  rotate,
+  scale
+} from '../src'
+
+describe('decompose', () => {
+  it('should not return transformations on identity decomposition', () => {
+    const decomposition = decompose(identity())
+    expect(decomposition).toMatchSnapshot()
```

Comment:
Snapshot testing is great but it is not enough accurate. Can you add some more tests that check the expected behavior verifying the returned object?